core/spec/models/spree/line_item_spec.rb

```diff
@@ -188,4 +188,12 @@
       expect(line_item.price).to eq 21.98
     end
   end
+
+  describe "precision of pre_tax_amount" do
+    let!(:line_item) { create :line_item, pre_tax_amount: 4.2051 }
+
+    it "keeps four digits of precision even when reloading" do
```

Comment:
@mamhoff we're very open to improvements in the taxation system (especially included tax, which is a bit of a mess). Is this refactor one PR or commit in particular?